businesscentral/image.yaml

```diff
@@ -0,0 +1,26 @@
+name: "jboss-bpmsuite-7/bpmsuite70-businesscentral"
+version: "7.0.0"
+from: "jboss-eap-7/eap70:latest"
+user: 185
+envs:
+    information:
+        - name: "JBOSS_PRODUCT"
+          value: "bpmsuite"
+        - name: "JBOSS_BPMSUITE_VERSION"
+          value: "7.0.0.CAP01"
```

Comment:
I'm not sure about this. We're targeting GA here and most probably this image won't be released/used before going GA. Additionally - I'm pretty sure the server itself when booting is advertising different version. Am I right?